sdcm/tester.py

```diff
@@ -466,13 +461,24 @@ def create_cluster(db_type='scylla'):
                     ec2_ami_id=self.params.get('ami_id_db_cassandra').split(),
                     ec2_ami_username=self.params.get('ami_db_cassandra_user'),
                     **cl_params)
+            elif db_type == 'mixed_scylla':
+                cluster.Setup.mixed_cluster(True)
+                cl_params.update(dict(ec2_instance_type=self.params.get('instance_type_db_oracle'),
+                                      user_prefix=user_prefix + '-oracle'))
+                return ScyllaAWSCluster(
+                    ec2_ami_id=self.params.get('ami_id_db_scylla_oracle').split(),
+                    ec2_ami_username=self.params.get('ami_db_scylla_user'),
+                    **cl_params)
 
         db_type = self.params.get('db_type')
         if db_type in ('scylla', 'cassandra'):
             self.db_cluster = create_cluster(db_type)
         elif db_type == 'mixed':
             self.db_cluster = create_cluster('scylla')
             self.cs_db_cluster = create_cluster('cassandra')
+        elif db_type == 'mixed_scylla':
+            self.db_cluster = create_cluster('scylla')
+            self.cs_db_cluster = create_cluster('mixed_scylla')
```

Comment:
So here, we are using scylla cluster to simulate c* cluster?
We may run gemini test by `two scylla clusters` or `one scylla cluster + one C cluster`?

Reply:
Yes, currently gemini verifies scylla vs scylla stable version with cache disabled, but actually it can be cassandra or something else - it just has to be stable